lib/bento_actionmailer.rb

```diff
@@ -5,6 +5,8 @@
 require "uri"
 
 module BentoActionMailer
+  # BentoActionMailer is an ActionMailer provider for sending mail through
```

Comment:
This comment is a bit verbose. But I would defer to @jessehanley to code styling questions.

Reply:
Will shorten! Appreciate the feedback - anything goes for improving the craft! 😁 🙏